BasePrepatcher.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Mono.Cecil;
+
+namespace SilksongPrepatcher;
+
+public abstract class BasePrepatcher
```

Comment:
Wonder if we should pull logger setup into here

Reply:
Yeah, I think this is a leftover from when I had a separate preloader patcher (as in a separate static class conforming to the BepInEx PP contract) for each preloader, I will change